src/configuration/app-containers.md

```diff
@@ -254,35 +255,35 @@ web:
             ...
 ```
 
-It has a few subkeys, which are:
+It has a few subkeys listed below:
 
 * `root`:
-    The folder from which to serve static assets for this location,
-    relative to the application root. The application root is the directory in
-    which the `.platform.app.yaml` file is located.  Typical values for this
+    The folder from which to serve static assets for this location
+    relative to the application root. The application root is the directory
+    where the `.platform.app.yaml` file is located.  Typical values for this
```

Comment:
Subjective: I think "in which" works better in context here.  The comma could go either way.